packages/react/src/components/CalendarDayGrid/CalendarDayGrid.styles.ts

```diff
@@ -149,6 +149,7 @@ export const styles = (props: ICalendarDayGridStyleProps): ICalendarDayGridStyle
     daySelected: [
       dateRangeType !== DateRangeType.Month && {
         backgroundColor: palette.neutralLight + '!important',
+        border: '1px solid #616161',
```

Comment:
is it possible that we have a palette color for this so that it isn't hard coded?

Reply:
Just made the change :)